### ISSUE TITLE
more visualization options

### ISSUE THREAD

**Issue by jasmainak** [autoreject/autoreject#204]
I see more and more that folks are just looking at the numbers `autoreject` throws out without looking at their data. I think we should emphasize this a bit and add some examples to show that. Here is a code snippet that could be added:

```py
epochs_orig = epochs.copy()
reject = get_rejection_threshold(epochs)

print('The rejection dictionary is %s' % reject)
epochs.drop_bad(reject=reject)

drop_log = [epochs.drop_log[s] for s in epochs.selection]
bad_idx = [idx for idx, ch_log in enumerate(drop_log) if ch_log != ()]
epochs_orig[bad_idx].plot()
```

Unfortunately, the sample data does not have any bad epochs. So we might have to use another dataset. Now that we have a dependency on `openneuro-py` for the examples (cc @alexrockhill @hoechenberger ;-)), I think we can make another nice example from the [code that was shared on the mailing list](https://mne.discourse.group/t/autorejection-on-resting-state-eeg-data/2998/5?u=mainakjas). Bonus points if we can have something like [`plot_overlay`](https://mne.tools/stable/generated/mne.preprocessing.ICA.html#mne.preprocessing.ICA.plot_overlay) in ICA

Here is a proposed API:

```py
from autoreject.viz import plot_overlay
plot_overlay(epochs, reject)
```

and it would show something similar to:

![image](https://user-images.githubusercontent.com/15852194/115898056-1e125900-a42b-11eb-862b-2dbe11d3c5c1.png)

but maybe with `evoked` instance. Unfortunately I don't have a lot of time right now but if someone takes the lead, I am happy to review!

**Comment by alexrockhill** [comment]
The openneuro dataset @apoorva6262 was working with https://openneuro.org/datasets/ds002778 might be a good one to add, the eeg files are fairly small and I'm sure at least one subject has lots of artifact contamination.

@apoorva6262 do you know which subject has a lot of artifacts? I can do this and you can review or if you want to take a shot you could try instead.

**Comment by jasmainak** [comment]
Yes, +1 on using `ds002778` from my end. Even using the script that @apoorva6262 shared would be a good start

**Comment by apoorva6262** [comment]
@alexrockhill , probably pd14. I think that one would be nice to show with (and without) ICA and Autoreject.

**Comment by jasmainak** [comment]
closed by #221
